CHANGELOG.md

```diff
@@ -1,5 +1,67 @@
 # Changelog
 
+## v0.111.0
+
+## Unreleased
+
+This Splunk OpenTelemetry Collector release includes changes from the opentelemetry-collector v0.111.0 and the opentelemetry-collector-contrib v0.111.0 releases where appropriate.
+
+### 🛑 Breaking changes 🛑
+
+
+- (Core) componentstatus: Remove deprecated functions ([#11363](https://github.com/open-telemetry/opentelemetry-collector/pull/11363))
+
+- (Core) service: remove deprecated types ([#11361](https://github.com/open-telemetry/opentelemetry-collector/pull/11361))
+
+
+### 🚩Deprecations 🚩
+
+- (Core) Remove deprecated funcs from processor package ([#11368](https://github.com/open-telemetry/opentelemetry-collector/pull/11368))
+
+- (Core) Deprecate and remove processorhelper.NewUnhealthyProcessorCreateSettings ([#11376](https://github.com/open-telemetry/opentelemetry-collector/pull/11376))
+
+- (Splunk) Remove ballast extension entirely from components  ([#5429](https://github.com/signalfx/splunk-otel-collector/pull/5429))
+
+- (Splunk) Deprecate jaegergrpc monitor ([#5428](https://github.com/signalfx/splunk-otel-collector/pull/5428))
+
+- (Splunk) Remove ballast extension entirely from components  ([#5429](https://github.com/signalfx/splunk-otel-collector/pull/5429))
+
+
+### 💡 Enhancements 💡
+
+- (Core) `mdatagen`: Generate documentation for components with resource attributes only ([#11242](https://github.com/open-telemetry/opentelemetry-collector/pull/11242))
+
+- (Core) `confignet`: Add Profiles Marshaler to otlptext. ([#11161](https://github.com/open-telemetry/opentelemetry-collector/pull/11161))
+
+- (Contrib) `cmd/opampsupervisor:` Add passthrough logging for collector ([#35474](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33980))
+
+- (Contrib) `receiver/awss3receiver:` Add ingest progress notifications via OpAMP ([#33980](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33980))
+
+- (Contrib) `exporter/clickhouse:` Add the ability to override default table names for all metric types ([#34251](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34251))
+
+- (Contrib) `otelarrowexporter:` Adjust configuration based on experimental results ([#35478](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35478))
+
+- (Contrib) `processor/logdedup:` feat: add ottl condition to logdedup processor ([#35443](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35443))
```

Comment:
These should be removed. We need only items for componets that are provided with our distro, see https://github.com/signalfx/splunk-otel-collector/blob/main/docs/components.md.